public/javascripts/jquery.tokeninput.js

```diff
@@ -729,11 +753,14 @@ $.TokenList = function (input, url_or_data, settings) {
         if (selected_dropdown_item) {
             $(selected_dropdown_item).removeClass(settings.classes.selectedToken);
         }
+        input_token.attr("aria-expanded", "false");
+        input_box.removeAttr("aria-activedescendant");
         selected_dropdown_item = null;
     }
 
     function show_dropdown() {
         dropdown
+            .attr("id", hidden_input_id + "_autocomplete_dropdown")
```

Comment:
We should use a var at the start of TokenList instead of doing ` + "_autocomplete_dropdown"` in 3 places.